subprojects/emcee/android-device/src/main/kotlin/com/avito/emcee/device/internal/Emulator.kt

```diff
@@ -0,0 +1,27 @@
+package com.avito.emcee.device.internal
+
+import com.avito.emcee.device.AndroidApplication
+import com.avito.emcee.device.AndroidDevice
+
+internal class Emulator : AndroidDevice {
```

Comment:
"emulator" looks like a leaky abstraction.
Should we know here about the implementation? Does a real device connected by adb differ?